src/R/data_splitting.R

```diff
@@ -0,0 +1,34 @@
+# author: Sam Zhang
+# date: 2022-03-31
+
+"This script splits a dataset into training/testing sets (75:25) and scales
+the training set so that all variables are on a comparable scale
+
+Usage: /src/R/data_splitting.R <file_path> <out_dir1> <out_dir2> <out_dir3>
+" -> doc
+
+library(tidymodels)
+library(tidyverse)
+library(docopt)
+
+set.seed(2022)
+
+opt <- docopt(doc)
+
+data <- read_csv(opt$file_path, show_col_types = FALSE)
+
+fire_split <- initial_split(data, prop = 0.75, strata = area)
+fire_train <- training(fire_split)
+fire_test <- testing(fire_split)
+
+fire_recipe <- recipes::recipe(area ~., data = fire_train) %>%
+  recipes::step_center(recipes::all_predictors()) %>%
+  recipes::step_scale(recipes::all_predictors())
+
+fire_scaled <- fire_recipe %>%
+  prep() %>%
+  bake(fire_train)
+
+write_csv(fire_train, paste0(opt$out_dir1, "/training_data.csv"))
+write_csv(fire_test, paste0(opt$out_dir2, "/testing_data.csv"))
+write_csv(fire_scaled, paste0(opt$out_dir3, "/scaled_and_centered_training_data.csv"))
```

Comment:
you can just use one command line argument! and update the usage to have one <out_dir>
```suggestion
write_csv(fire_train, paste0(opt$out_dir, "/training_data.csv"))
write_csv(fire_test, paste0(opt$out_dir, "/testing_data.csv"))
write_csv(fire_scaled, paste0(opt$out_dir, "/scaled_and_centered_training_data.csv"))
```